demo-project/src/demo_project/settings.py

```diff
@@ -9,7 +9,8 @@
 from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
 
 SESSION_STORE_CLASS = SQLiteStore
-SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
+SESSION_STORE_ARGS = {"local_path": str(Path(__file__).parents[2] / "data"),
```

Comment:
Does `local_path` get used anywhere? I don't see it 🤔 

We need to think about how to make this backwards compatible with argument names - will explain on Monday.